CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ Exceptions are acceptable depending on the circumstances (critical bug fixes tha
 - corrected the way the delivery and deployment steps are skipped or not in Azure DevOps
 - corrected GoLang delivery and deployment to have the default image and the proper format
 - changed Javascript deployment to continue tasks with error
+- corrected use of stageDependencies in the `deployment` stage
```

Comment:
Please, put the code quote in "stageDependencies"

Reply:
Done.